shell.nix

```diff
@@ -0,0 +1,36 @@
+with import (builtins.fetchTarball {
+    name = "24.05"; # May 31 2024
+    url = "https://github.com/NixOS/nixpkgs/archive/refs/tags/24.05.tar.gz";
+    sha256 = "sha256:1lr1h35prqkd1mkmzriwlpvxcb34kmhc9dnr48gkm8hh089hifmx";
+}) {};
+mkShell {
+  buildInputs =
+    let
+      ourPg = callPackage ./nix/postgresql {
+        inherit lib;
+        inherit stdenv;
+        inherit fetchurl;
+        inherit makeWrapper;
+        inherit callPackage;
+      };
+      supportedPgVersions = [
+        postgresql_13
+        postgresql_14
+        postgresql_15
+        postgresql_16
+        # ourPg.postgresql_17
```

Comment:
from nixpkgs or generally?
the Oriole preview is going out on pg17, is that impacted?

Reply:
This can be fixed with a one liner https://github.com/supabase/supautils/pull/97/files#diff-c2fdfbbb68a7de780726ceef2637dd3157eb70ac24ff69e7ebbcb835e9d1c24bR199

Reply:
Fixed, thanks!